js/app.js

```diff
@@ -9,4 +9,127 @@ hamburger.addEventListener('click', () => {
+    SeeSource: 'https://github.com/JuanDiegoGuerra/Portfolio-Setup-and-Mobile-first',
+  },
+  {
+    id: 'work4',
+    projectName: 'Caesar Cipher',
+    projectInfo:
+      'Caesar Cipher is a personal project that I made watching a video and practicing what I learnd in my Web Developer journey.',
+    ProjectIMG: [
+      'img/project_imgs/caesar_cipher_preview.gif',
+    ],
+    languages: ['CSS', 'HTML5', 'Github', 'Javascript'],
+    seeProject: 'See Project',
+    SeeLive: 'https://github.com/JuanDiegoGuerra/Portfolio-Setup-and-Mobile-first',
+    SeeSource: 'https://github.com/JuanDiegoGuerra/Portfolio-Setup-and-Mobile-first',
+  },
+  {
+    id: 'work5',
+    projectName: 'Chronometer',
+    projectInfo:
+      'Chronometer is a personal project that I made watching a video and practicing what I learnd in my Web Developer journey.',
+    ProjectIMG: [
+      'img/project_imgs/chronometer_preview.gif',
+    ],
+    languages: ['HTML5', 'Javascript', 'CSS', 'Github'],
+    seeProject: 'See Project',
+    SeeLive: 'https://github.com/JuanDiegoGuerra/Portfolio-Setup-and-Mobile-first',
+    SeeSource: 'https://github.com/JuanDiegoGuerra/Portfolio-Setup-and-Mobile-first',
+  },
+  {
+    id: 'work6',
+    projectName: 'Try Boostrap',
+    projectInfo:
+      'Try Boostrap is a project in where you built your own portfolio website using Boostrap.',
+    ProjectIMG: [
+      'img/project_imgs/try_boostrap.JPG',
+    ],
+    languages: ['Boostrap', 'HTML5', 'CSS', 'Github'],
+    seeProject: 'See Project',
+    SeeLive: 'https://github.com/JuanDiegoGuerra/Portfolio-Setup-and-Mobile-first',
+    SeeSource: 'https://github.com/JuanDiegoGuerra/Portfolio-Setup-and-Mobile-first',
+  },
+];
+// Pop Up section
+const PopupWindow = document.querySelectorAll('.external_link');
+const popUp = document.querySelector('.popup');
+
+PopupWindow.forEach((p, index) => {
+  p.addEventListener('click', () => {
+    document.body.classList.add('no-scroll');
+    popUp.classList.add('display-visible');
+    popUp.innerHTML = `
+  <div class="popup-container display-flex-column">
+        <div class="close-pop"><i class="fa fa-times" aria-hidden="true"></i></div>
+        <div class="slider">
+            <div class="pop-img-container">
+            <img id="project-img" class="project-img" src="${ProjectCardsDetails[index].ProjectIMG[0]}" alt="">
+            </div>
+        </div>
+        <h2 class="project-name work-name" id="workname-title">${ProjectCardsDetails[index].projectName}</h2>
+        <ul class="works-tools work-name">
+          <li class="work-tool">${ProjectCardsDetails[index].languages[0]}</li>
+          <li class="work-tool">${ProjectCardsDetails[index].languages[1]}</li>
+          <li class="work-tool">${ProjectCardsDetails[index].languages[2]}</li>
+          <li class="work-tool">${ProjectCardsDetails[index].languages[3]}</li>
+        </ul>
+        <p class="notes">
+        ${ProjectCardsDetails[index].projectInfo}
+        </p>
```

Comment:
- [ ] Personally, I think it is not a good idea of using a js array, it is efficient and easy to use the map method over it to get all its items then using indexes[0...], what if you have an array of 100 items, are you gonna do [0],[1],[n?]?   
use
``` 
project[index].map((item) => {
         <div> item.languages </div> 
#You can do the same for the rest of the items too
})
```